de.syping.gta5view.yaml

```diff
@@ -0,0 +1,26 @@
+app-id: de.syping.gta5view
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: gta5view
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+modules:
+  - name: gta5view
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DFLATPAK_BUILD=ON
+      - -DQCONF_BUILD=ON
+    sources:
+      - type: git
+        url: https://github.com/SyDevTeam/gta5view.git
+        tag: 1.8.0
+        commit: 547922329afbd8b35d63d81ea5772563a1c24fe2      
+      - type: file
+        path: de.syping.gta5view.appdata.xml
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html